boards/shared/Inc/App/App_SharedRgbLed.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+/**
+ * Allocate and initialize a RGB LED
+ * @param turn_red Turn the RGB LED red
+ * @param turn_green Turn the RGB LED green
+ * @param turn_blue Turn the RGB LED blue
+ * @param turn_off Turn off the RGB LED
+ * @return The created RGB LED, whose ownership is transferred to the caller
+ */
+struct RgbLed *App_SharedRgbLed_Create(
```

Comment:
do we need to pre-declare this struct?

Reply:
Oh yeah, not sure how this still compiles tbh 🙃